lib/storage/index.ts

```diff
@@ -84,7 +84,7 @@ export async function initializeStorage() {
           .values({
             created_at: new Date().toISOString(),
             driver_upload_id: driverUploadId,
-            id: uploadId,
+            id: uploadId.toString(),
```

Comment:
instead .toString() above?

Reply:
the `uploadId` returned by `reserveCache` needs to be a number

Reply:
Why? Just asking

Reply:
github's package expects a number